coreapps-web-2.2/pom.xml

```diff
@@ -0,0 +1,147 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+      <parent>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>coreapps</artifactId>
+        <version>1.32.0-SNAPSHOT</version>
+     </parent>
+     <artifactId>coreapps-web-2.2</artifactId>
+     <packaging>jar</packaging>
+     <name>coreapps Module web 2.2</name>
+     <description>2.2 web project for coreapps</description>
+     
+     <properties>
+        <openMRSVersion>2.5.0-SNAPSHOT</openMRSVersion>
+        <reportingVersion>0.10.3</reportingVersion>
+        <calculationVersion>1.3.0-SNAPSHOT</calculationVersion>
+     </properties>
+     
+     <dependencies>
+         <dependency>
+           <groupId>${project.parent.groupId}</groupId>
+           <artifactId>${project.parent.artifactId}-api</artifactId>
+           <version>${project.parent.version}</version>
+           <scope>provided</scope>
+         </dependency>
+         
+          <dependency>
+            <groupId>org.openmrs.api</groupId>
+            <artifactId>openmrs-api</artifactId>
+            <version>${openMRSVersion}</version>
+            <scope>provided</scope>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.openmrs.api</groupId>
+            <artifactId>openmrs-api</artifactId>
+            <type>test-jar</type>            
+            <version>${openMRSVersion}</version>
+            <scope>test</scope>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.openmrs.test</groupId>
+            <artifactId>openmrs-test</artifactId>
+            <version>${openMRSVersion}</version>
+            <type>pom</type>
+            <scope>test</scope>
+        </dependency>
+        
+        <dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>emrapi-api</artifactId>
```

Comment:
@HerbertYiga  add the other emrapi submodule dependencies as they're added in the [Parent POM](https://github.com/openmrs/openmrs-module-coreapps/blob/master/pom.xml#L117-L152)  and see if atleast it solves your problem